Userland/Services/WindowServer/WindowManager.cpp

```diff
@@ -419,6 +419,14 @@ void WindowManager::greet_window_manager(WMConnectionFromClient& conn)
     });
     if (auto* applet_area_window = AppletManager::the().window())
         tell_wms_applet_area_size_changed(applet_area_window->size());
+    tell_wm_greeting_is_over(conn);
+}
+
+void WindowManager::tell_wm_greeting_is_over(WMConnectionFromClient& conn)
+{
+    if (conn.wm_id() < 0)
+        return;
+    conn.async_greeting_is_over(conn.wm_id());
```

Comment:
This isn't really "dead code", but it looks like it doesn't serve a purpose yet.

Reply:
It does, that's used in winlist

Reply:
I mean, yeah, in that sense it is related, but not to the commit that adds the rect setting API itself, especially since the other half of the `greeting_is_over` handling only gets introduced in a later commit.

Reply:
no idea how it ended up in that commit
i wonder if i will ever be good at using git someday 😩